docs/guides/LiveCode Builder Language Reference.md

```diff
@@ -577,7 +577,13 @@ on Android and iOS).
 
 The Obj-C binding string has the following form:
 
-    "objc:[class.](+|-)method[?thread]"
+    "objc:[library>][class.](+|-)method[?thread]"
+
+Here *library* specifies the name of the library or framework to bind to (if no
+library is specified a symbol from the engine executable or a library it is
+linked to is assumed). If binding to a system framework include `.framework` in
+the library name, however, if binding to a framework in code resource folder do
+not include the extension.
```

Comment:
Hmm... looking at the native path loading stuff again I think I'm wrong here and adding `.framework` is not necessary for system frameworks. I will test that to make sure.

Reply:
OK so when `s_load_library_callback` is set then we try lots of different extensions in `MCU_library_load` but otherwise we just try the library name which means it needs the `.framework` included. I'm wondering if we should move the code that tries adding different extensions into `MCSLibraryCreateWithPath` for consistency? If we do that then whether framework or not or whether running under the engine or `lc-run` then you just use the name of the library/framework/bundle.

Reply:
See https://github.com/livecode/livecode/pull/6669 for ^